ui/apps/platform/tailwind.config.js

```diff
@@ -20,4 +20,5 @@ module.exports = {
             '6xl': '2.5rem', // 40px
         },
     },
+    purge: ['./src/**/*.{js,ts,tsx}'],
```

Comment:
I haven't spent much time investigating how Tailwind is set up in out project, but is there any chance this purges classes that are defined only in `packages/ui-components/*` and not in the main app? The scope of the directory that purge is looking for seems to be limited to `apps/platform/src`.

Reply:
Good question and good timing. I am investigating at this moment

Here are the elements from ui-components:
* 16 `Message` we can replace with PatternFly `Alert`
* 5 `DetailedTooltipOverlay` and `HoverHint` to be determined which are essential
* 1 `Avatar` to be determined
* 1 each of `Button` and `CondensedButton` in WatchedImagesDialog

I can search to verify the likely situation that the only classes they need are also in ui/src folder.

The most direct way to future proof the purge is to remove import from ui-components in the 19 files ui files.

Reply:
By the way, Gavin removed from infra the few import from ui-component last week.

Reply:
Okay, this pull request needs to wait until we replace from ui-components. Which becomes a priority.

Analysis of 75 unique classes in the elements that ui/src imports from ui-components:
* 53 classes also used in ui/src although some have only one occurrence
* 2 classes defined in ui-components.css file
* 4 classes defined nowhere :)
* 1 class used in `DetailedToolTipOverlay` **but not** in ui/src
* 5 classes used in `Message` **but not** in ui/src
* 10 classes used in `Button` or `CondensedAlertButton` **but not** in ui/src